ichnaea/conftest.py

```diff
@@ -82,14 +82,6 @@ def monkeysession(request):
 
 @pytest.fixture(scope="session", autouse=True)
 def package():
-    # We do this here as early as possible in tests.
-    # We only do it in tests, as the real celery processes should
-    # run without the monkey patches applied. The gunicorn arbiter
-    # patches its worker processes itself.
-    from gevent import monkey
-
-    monkey.patch_all()
```

Comment:
After upgrading gunicorn and gevent, this kicks up an error due to a warning gevent added where it checks a bunch of modules to see if they were loaded before monkey patching happened.

I took this out for a couple of reasons. First, it seems to suggest we can just monkey patch the webapp tests, but that doesn't fly--it patches everything for everything. Second, it kicks up the error.

If we really want to run the webapp tests monkey patched, they should be a completely separate test suite that we run separately that monkey patches earlier in the test run. Doing what we're doing now has undefined behavior.